app/build.gradle.kts

```diff
@@ -0,0 +1,173 @@
+import java.util.Properties
+import java.io.FileInputStream
+
+plugins {
+    id("com.android.application")
```

Comment:
`alias` here too 😉

Reply:
Ah ok I see!

Reply:
I've replaced most of the `id` by `alias` but I do not know how to do it for hilt without breaking everything. I get strange errors about `android` being undefined. Could you have a look at it? Thanks.